pkg/tests/upgrade_tests/upgrade_test.go

```diff
@@ -86,23 +86,12 @@ func getDBImages(extensionState testhelpers.TestOptions, prevPromscaleVersion *s
 		panic("Only use pg12 for upgrade tests")
 	}
 
-	// From Promscale 0.14.0 onwards the minimum extension version is 0.6.0
+	// From Promscale 0.15.0 onwards the minimum extension version is 0.7.0
 	if prevPromscaleVersion != nil && prevPromscaleVersion.GE(semver.MustParse("0.14.0")) {
-		return "timescale/timescaledb-ha:pg" + pgVersion + ".12-ts2.7.2-latest", dockerImageName, nil
+		return "timescale/timescaledb-ha:pg" + pgVersion + ".12-ts2.8.1-latest", dockerImageName, nil
 	}
 
-	// From Promscale 0.13.0 onwards the minimum extension version is 0.5.4
-	if prevPromscaleVersion != nil && prevPromscaleVersion.GE(semver.MustParse("0.13.0")) {
-		return "timescale/timescaledb-ha:pg" + pgVersion + ".11-ts2.7.1-latest", dockerImageName, nil
-	}
-
-	// From Promscale 0.11.0 onwards the minimum extension version is 0.5.0
-	if prevPromscaleVersion != nil && prevPromscaleVersion.GE(semver.MustParse("0.11.0")) {
-		return "timescale/timescaledb-ha:pg" + pgVersion + "-ts2.6-latest", dockerImageName, nil
-	}
-
-	//return "timescaledev/promscale-extension:0.1.2-ts2-pg" + pgVersion, dockerImageName, nil
```

Comment:
We don't need to handle older version other than previous because thats what we test.